installation/install.sh

```
@@ -2076,6 +2078,49 @@ is_install_virtualenv(){
+    if [ $? -ne 0 ]; then
+        echo "key-manager package not found in repo(s) $ZSTACK_YUM_REPOS, skip install" >>$ZSTACK_INSTALL_LOG
+        pass
+        return
+    fi
+
+    echo yum install --disablerepo="*" --enablerepo="$ZSTACK_YUM_REPOS" -y key-manager >>$ZSTACK_INSTALL_LOG
+    yum install --disablerepo="*" --enablerepo="$ZSTACK_YUM_REPOS" -y key-manager >>$ZSTACK_INSTALL_LOG 2>&1
+    [ $? -ne 0 ] && fail "failed to install key-manager from repo(s) $ZSTACK_YUM_REPOS"
+
+    if systemctl list-unit-files crypto-daemon.service 2>/dev/null | grep -q '^crypto-daemon.service'; then
+        systemctl enable crypto-daemon.service >>$ZSTACK_INSTALL_LOG 2>&1
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)